src/main/java/net/dries007/tfc/api/types/Metal.java

```diff
@@ -280,7 +285,7 @@ public static Item create(Metal metal, ItemType type)
 
         public boolean hasType(Metal metal)
         {
-            if (!metal.usable)
+            if (!metal.isUsable())
```

Comment:
No need to change this, we're encapsulated in the same class anyways.

Reply:
it's for cases where a class extends Metal and usability differs from what is passed to the constructor

Reply:
Would like to note that it is more consistent with this change.
[Line 299 does something similar](https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/1.12.x/src/main/java/net/dries007/tfc/api/types/Metal.java#L299)

Reply:
I agree with Charnuz, it should match the function for consistency and for children classes.

Reply:
I agree with Charnuz, this should match the function for children classes.